Managers/LuaMan.cpp

```diff
@@ -2194,10 +2194,11 @@ int LuaMan::Create()
 
         // NOT a member function, so adopting _1 instead of the _2 for the first param, since there's no "this" pointer!!
         def("DeleteEntity", &DeleteEntity, adopt(_1)),
-        def("PosRand", &PosRand),
+		def("RangeRand", (double(*)(double, double))& RandomNum),
+        //def("PosRand", (double(*)(void))& RandomNum),
```

Comment:
What's up with this one?

Reply:
Just a documentation of what I already tried. I'll delete it.